lib/map-generator.ts

```diff
@@ -244,7 +252,7 @@ export default class MapGenerator {
   private toJPEG(canvas: HTMLCanvasElement, fileName: string) {
     const uri = canvas.toDataURL('image/jpeg', 0.85);
     // @ts-ignore
-    if (canvas.msToBlob) {
+    if (canvas.msToBlob && window.navigator?.msSaveBlob) {
```

Comment:
@Simon25500 To be honest, I just confirmed my typescript also shows error at `window.navigator.msSaveBlob(blob, fileName);`...

I think we can simply remove this code for IE11 since IE11 is no longer supported officially by Microsoft. What do you think?

Reply:
@JinIgarashi ah great, I though I was the only one to have this errors ;) but yes I think we can do that !

Reply:
@JinIgarashi I made the modification